eth_tester/main.py

```diff
@@ -285,6 +326,10 @@ def get_transaction_receipt(self, transaction_hash):
     #
     def enable_auto_mine_transactions(self):
         self.auto_mine_transactions = True
+        sent_transaction_hashes = [self.send_transaction(extract_valid_transaction_params(tx))
```

Comment:
Can we be strict here and _reject_ transactions that have invalid transaction params?

Reply:
I think this is too much of un unlikely scenario to be handled. These transactions are added to the list from `get_transaction_by_hash` which _should_ be valid. The only real purpose of this extraction is to not send the params like v,r,s from the dict that is returned by `get_transaction_by_hash`.

We could add validation when we add the transaction to the list but I don't really see the purpose.

Reply:
> These transactions are added to the list from `get_transaction_by_hash`

Fair enough, I'm okay skipping this validation.